### DISCUSSION TITLE
2009/05/24/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E8%A1%A5%E5%85%85%E5%86%85%E5%AE%B9%E5%92%8C%E6%B3%A8%E9%87%8A

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#266] [Announcements]
# 2009/05/24/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E8%A1%A5%E5%85%85%E5%86%85%E5%AE%B9%E5%92%8C%E6%B3%A8%E9%87%8A

最新的内容都在这里了：https://lv.qizhen.xyz/

https://blog.qizhen.xyz/2009/05/24/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E8%A1%A5%E5%85%85%E5%86%85%E5%AE%B9%E5%92%8C%E6%B3%A8%E9%87%8A

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Di
2009年07月7日 03:24 编辑
阮大哥请教一下，我现在用的用的Lecroy公司的示波器，Waverunner 6100A。做了一个Parameter 1，是接口1的信号的rise time，但是不能一直不断向示波器申请每个数据，会造成无响应，所以做了个function是存储的parameter 1的trend，但是问题是如何可以把这个F1的数据用labview通过远程控制命令读取出来，在官方的命令表中没有找到需要的，还请指导一下，谢谢！

回复
NeoYu
2009年05月27日 13:37 编辑
刚才在群里Ian说要做新加坡和马来西亚的唯一分销商，还要给你翻译英文呢，走向世界，我看这主意不错，

回复
NeoYu
2009年05月27日 13:36 编辑
另外，可不可以提起透露一下此书的价格？

回复
NeoYu
2009年05月27日 13:33 编辑
想请问阮大侠主要对LabVIEW的哪些方面比较深入，建议对您比较深入的使劲写写，：）当然，小技巧也很有意思，说实话，经常对您所分享出来的小技巧感到非常吃惊，谢谢。希望看到这本书时能够有个大大的惊喜，再次谢谢。

回复
